modules/claims_api/config/schemas/v2/526.json

```diff
@@ -871,14 +872,18 @@
                           "additionalProperties": false,
                           "required": ["areaCode", "phoneNumber"],
                           "properties": {
-                            "areaCode": {
-                              "type": "string",
-                              "maxLength": 3
-                            },
-                            "phoneNumber": {
-                              "type": "string",
-                              "maxLength": 7
-                            }
+                              "areaCode": {
+                                "type": "string",
+                                "maxLength": 3,
+                                "pattern": "^\\d{3}$",
+                                "example": "688"
+                              },
```

Comment:
Kayla wants the area code example to be consistent, as "555", and the phone # to be 5555555. There is a ticket for this, we can remove it or change it, up to you.

Reply:
ok I can add that in here on all these